independent-projects/bootstrap/gradle-plugin/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,9 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+<<<<<<< HEAD
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.9-all.zip
+=======
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.8-bin.zip
+>>>>>>> d232aad03c (Add wrapper)
```

Comment:
This looks like a leftover from a merge?

Reply:
Oh right, this is a mistake.